audit/projects/k8s-release/buckets/k8s-release/iam.json

```diff
@@ -10,7 +10,8 @@
     },
     {
       "members": [
-        "projectViewer:k8s-release"
+        "projectViewer:k8s-release",
+        "serviceAccount:project-304687256732@storage-transfer-service.iam.gserviceaccount.com"
```

Comment:
Thanks for the free unannounced IAM change https://cloud.google.com/storage-transfer

Reply:
https://cloud.google.com/storage-transfer/docs/configure-access

>Storage Transfer Service uses a Google-managed service account to move your data. If you create a transfer from Google Cloud Console and have permissions to update IAM policies for Cloud Storage resources, then transfers created from Google Cloud Console automatically grant the Google-managed service account used by Storage Transfer Service the required permissions for the transfer.

So that's what happened when I did https://github.com/kubernetes/k8s.io/issues/1569#issuecomment-877454553

Reply:
You know what's neat? I can't actually find the code that provisions this bucket

Reply:
```
gsutil iam ch -d serviceAccount:project-304687256732@storage-transfer-service.iam.gserviceaccount.com:roles/storage.legacyBucketReader gs://k8s-release
```